docs/master/performance/caching.md

```diff
@@ -102,6 +102,15 @@ type GithubProfile {
 }
 ```
 
+## Implementing your own cache key generator
+
+In laravel `AppServiceProvider.php`, bind the [`Nuwave\Lighthouse\Support\Contracts\CacheKeyAndTags.php`](https://github.com/nuwave/lighthouse/blob/master/src/Support/Contracts/CacheKeyAndTags.php) interface to your cache key generator class:
+
+```php
+$this->app->bind(CacheKeyAndTags::class, YourOwnCacheKeyGenerator::class);
+```
+You can extend lighthouse [`Nuwave\Lighthouse\Cache\CacheKeyAndTagsGenerator.php`](https://github.com/nuwave/lighthouse/blob/master/src/Cache/CacheKeyAndTagsGenerator.php) class and override the methods or implement the interface.
```

Comment:
```suggestion
```

You can extend [`Nuwave\Lighthouse\Cache\CacheKeyAndTagsGenerator.php`](https://github.com/nuwave/lighthouse/blob/master/src/Cache/CacheKeyAndTagsGenerator.php)
to override certain methods, or implement the interface from scratch.
```